### ISSUE TITLE
Screen Reader: Track Distance: Announced instructions to reorder map start/stop points do not work

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76834]
## Action Performed:
Prerequisites:
- The user is signed in.
- The start and stop points have been selected on the track distance page.

1. Using Chrome + JAWS, open the URL - https://new.expensify.com/
2. Navigate to the "+" icon and press Enter
3. Tab to the "Track distance" menu item and press Enter to open the Pop-up
4. Navigate and select 'Start' map points and select any address for the start point.
5. Navigate and select 'Stop' map points and select any address for the start point.
6. Listen to the instructions announced for reordering
7. Try to reorder the point using the keys mentioned in the announcement

## Expected Result:
Users must be able to reorder the 'start/stop' points on the map using the keys mentioned in the announcement, so that the user can change the position of a selected waypoint in the route list.

## Actual Result:
When focus moves to the 'Start/stop point, the screen reader announces the instruction as 'Sort, but the described keyboard actions do not change the order; so that the instruction provided using the screen reader creates misleading instructions for the users

HTML:
<div aria-label="" role="menuitem" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-17gur6a r-18u37iz r-1wtj0ep r-1mmae3n r-u9wvl5 r-13qz1uu" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx" style=""><div class="css-g5y9jx r-13awgt0"><div class="css-g5y9jx r-18u37iz"><div class="css-g5y9jx r-eqz5dr r-13awgt0"><div class="css-g5y9jx r-18u37iz r-105ug2t"><div class="css-g5y9jx r-1awozwy r-1777fci" style="width: 40px;"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22)
xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#8B9C8F"><path d="M1 6c0-1.1.9-2 2-2h14c1.1 0 2 .9 2 2s-.9 2-2 2H3c-1.1 0-2-.9-2-2m0 8c0-1.1.9-2 2-2h14c1.1 0 2 .9 2 2s-.9 2-2 2H3c-1.1 0-2-.9-2-2"></path></svg></div></div><div class="css-g5y9jx r-1awozwy r-1777fci r-1aockid"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22)
xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#8B9C8F"><path d="M10 15.5c3 0 5.5-2.5 5.5-5.5S13 4.5 10 4.5 4.5 7 4.5 10 7 15.5 10 15.5m0 2c4.1 0 7.5-3.4 7.5-7.5S14.1 2.5 10 2.5 2.5 5.9 2.5 10s3.4 7.5 7.5 7.5" style="fill-rule: evenodd; clip-rule: evenodd;"></path></svg></div></div><div class="css-g5y9jx r-1777fci r-13awgt0" style="min-height: 40px;"><div dir="auto" class="css-146c3p1 r-8akbws r-krxsd3 r-dnmrzs r-1qsk4np r-1udbk01" style="-webkit-line-clamp: 2; color: rgb(175, 187, 176); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; margin-left: 12px; line-height: 20px; word-break: break-word;">Start</div></div></div></div><div class="css-g5y9jx r-18u37iz" style="min-height: 40px;"><div class="css-g5y9jx r-105ug2t" style="justify-content: center; align-items: center; width: 40px;"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22)
xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#8B9C8F"><path d="M5.6 18.4c.8.8 2 .8 2.8 0l8.4-8.4-8.4-8.4c-.8-.8-2-.8-2.8 0s-.8 2 0 2.8l5.6 5.6-5.6 5.6c-.8.8-.8 2 0 2.8" style="fill-rule: evenodd; clip-rule: evenodd;"></path></svg></div></div></div></div></div></div>
y9jx r-<div>(...)<div>

## Area issue was found in:
Track Distance

## Failed WCAG checkpoints
WCAG 3.3.2 Labels or Instructions
https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions

## User impact:
Providing clear and unambiguous labels and instructions (including examples of expected data formats) helps all users - but particularly those with cognitive, language, and learning disabilities - to enter information correctly and can prevent users from making incomplete or incorrect form submissions.

## Suggested resolution:
Ensure that the instructions that are announced by the screen reader actually work with the keyboard, so that the users can reorder the points
See some reordering examples here: https://salesforce-ux.github.io/dnd-a11y-patterns/#/?_k=tfo2nc

An alternative is to use additional buttons for each entry to move it up or down.

## Environment(s):

https://github.com/user-attachments/assets/37a4512b-bf2f-4673-ba89-e270c0e82b9a

<img width="1919" height="970" alt="Image" src="https://github.com/user-attachments/assets/dde3fea2-0640-40ba-a59d-5f2e717ffb94" />

Windows Windows 11 Home - Chrome

## Screenshots/Videos

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6996968
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Issue Owner</summary>Current Issue Owner: @eVoloshchak</details>

**Comment by rushatgabhane** [comment]
Removed A11y label because there exists legit workaround to edit waypoints 

https://expensify.enterprise.slack.com/archives/C9YU7BX5M/p1774495911936519?thread_ts=1773667468.450889&cid=C9YU7BX5M

**Comment by eVoloshchak** [comment]
Same as above

**Comment by eVoloshchak** [comment]
https://github.com/Expensify/App/pull/85474 is close to being merged

**Comment by eVoloshchak** [comment]
Same as above

**Comment by eVoloshchak** [comment]
https://github.com/Expensify/App/pull/85474 is close to being merged

**Comment by eVoloshchak** [comment]
Not overdue, https://github.com/Expensify/App/pull/85474 is in the works

**Comment by eVoloshchak** [comment]
Same as above

**Comment by eVoloshchak** [comment]
https://github.com/Expensify/App/pull/85474 is in progress

**Comment by eVoloshchak** [comment]
https://github.com/Expensify/App/pull/85474 is in progress

**Comment by rushatgabhane** [comment]
@iwiznia all yours - https://github.com/Expensify/App/pull/85474

**Comment by eVoloshchak** [comment]
Not overdue, https://github.com/Expensify/App/pull/85474 is merged!

**Comment by eVoloshchak** [comment]
Not overdue, https://github.com/Expensify/App/pull/85474 is deployed to prod 5 days ago

**Comment by eVoloshchak** [comment]
This doesn't require payment, BZ Checklist is at https://github.com/Expensify/App/issues/79247#issuecomment-3817394676

**Comment by eVoloshchak** [comment]
Not overdue, this was resolved and can be closed

**Comment by eVoloshchak** [comment]
Not overdue, this was resolved and can be closed
